code-name-taran/src/com/spaghettisoft/component/chess/figures/IFigures.java

```diff
@@ -0,0 +1,11 @@
+package com.spaghettisoft.component.chess.figures;
+
+import com.spaghettisoft.component.chess.players.Move;
+
+public interface IFigures {
+	
+	boolean validMove (Move move);

```

Comment:
The boolean variable names should usually express a question. A better name here will be "isValidMove".